python/pytest.ini

```diff
@@ -1,4 +1,5 @@
 [pytest]
 markers =
     smoke_test: mark a test as a build verification testing.
+    skip_version: parametrized mark for fast skipping tests by server version.
```

Comment:
Maybe a better name like `min_version` or `required_version` ? Or even `skip_below_version`

Reply:
we might want include a similar tag for deprecated commands, like `deprecated_in_version`

Reply:
Renamed